services/director/src/simcore_service_director/config.py

```diff
@@ -60,6 +61,28 @@ def _from_env_with_default(env: str, python_type, default):
     "DIRECTOR_SERVICES_CUSTOM_CONSTRAINTS", ""
 )
 
+
+def _parse_placement_replacements() -> Optional[Dict[str, str]]:
+    result: Optional[Dict[str, str]] = None
+    str_env_var: Optional[str] = os.environ.get(
+        "DIRECTOR_PLACEMENT_CONSTRAINTS_REPLACEMENTS_FOR_GENERIC_RESOURCES", None

```

Comment:
`DIRECTOR_PLACEMENT_CONSTRAINTS_REPLACEMENTS_FOR_GENERIC_RESOURCES=null` will not work here. it will return a string that is not json.load-able

Reply:
this got changed as well